apps/gallery/js/startup.js

```diff
@@ -273,11 +273,17 @@
       window.performance.mark('contentInteractive');
 
       // Remove the attribute to trigger gaia header font-fit logic, defer it
-      // til now to postpone the overhead.
-      var headers = document.querySelectorAll('gaia-header');
-      for (var i = 0; i < headers.length; ++i) {
-        headers[i].removeAttribute('no-font-fit');
-      }
+      // til now to postpone the overhead. Add setTimeout to lazyload
+      // gaia-header.js aynchronously.
+      setTimeout(function() {
```

Comment:
Thats what I thought , but on double check I found https://bugzilla.mozilla.org/show_bug.cgi?id=926851 still open , and till its resolved we need setTimeout to make it async.

While testing, I tried to tap on select as soon as its visible and still see gaia-header loaded. I guess its too fast to catch the interaction manually. I can  move performance.mark('contentInteractive') after the  select and thumbnail event handler are registered and will post updated numbers.